### PR TITLE
Add Dollar General

### PR DESCRIPTION
**Pull request by Rabadash8820**
Add dollargeneral.com, for the Dollar General store.

This dialog shows when creating a Dollar General online account. A phone number is required, and is then used to send SMS verification codes on every login. This step cannot be skipped.

image

### CODE REVIEW

File: entries/d/dollargeneral.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Dollar General": {
+    "domain": "dollargeneral.com",
+    "img": "dollargeneral.com.png",
+    "tfa": [
+      "sms"
+    ],
+    "categories": [
+      "retail"
+    ]
```
Comment (hkamran80) [inline entries/d/dollargeneral.com.json:15]:
```suggestion
    ],
    "regions": [
      "mx",
      "us"
    ]
```

File: entries/d/dollargeneral.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Dollar General": {
+    "domain": "dollargeneral.com",
+    "img": "dollargeneral.com.png",
+    "tfa": [
+      "sms"
```
Comment (hkamran80) [inline entries/d/dollargeneral.com.json]:
```suggestion
      "sms",
      "call"
```

Your screenshot says that calls might also be used.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/d/dollargeneral.com.json (hunk 2 above)
Comment (Rabadash8820) [reply entries/d/dollargeneral.com.json]: Good catch

File: entries/d/dollargeneral.com.json (hunk 1 above)
Comment (Rabadash8820) [reply entries/d/dollargeneral.com.json:15]: Also good catch!

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**